modules/ROOT/pages/installation/osx.adoc

```diff
@@ -20,26 +20,28 @@ Select the appropriate tar.gz distribution for your platform.
 For example, `tar -xf neo4j-community-{neo4j-version-exact}-unix.tar.gz`.
 . Place the extracted files in a permanent home on your server.
 The top level directory is referred to as NEO4J_HOME.
-. From Neo4j v5.4 onwards, you are required to accept the license agreement before running the Neo4j Enterprise Edition.
+. From Neo4j v5.4 onwards, you are required to accept either the commercial or the evaluation license agreement before running the Neo4j Enterprise Edition.
 If you are using Community Edition, you can skip this step.
-.. Use one of the following options to accept the commercial license agreement.
+* Use one of the following options to accept the commercial license agreement.
 See https://neo4j.com/terms/licensing/ for more information.
```

Comment:
I don't know. It's a link outside the scope of the docs. Usually, when it is an external link, we provide some context so that it is clear from just seeing it that it's an external link. So, `link:https://neo4j.com/terms/licensing/[Licensing]` won't work. Do you have any idea how to refer to this link?

Reply:
Well it's not exactly external because it's still neo4.com 🤔

Reply:
Yeah, but it's external for the Operations manual. Usually, we write something like **Cypher Manual -> the section**. But here I am not sure what to write.

Reply:
Maybe something like "Neo4j licensing" (even though the page itself is just Licensing?

Reply:
Hmm, when you open that link, you see a list of licenses per location, not the agreement itself. So maybe "See the link:https://neo4j.com/terms/licensing/[Neo4j licensing] page for details on the available agreements." 😄

Reply:
I think that can do!

Reply:
Ok, cool! Thanks, @lidiazuin!